UITests/UITests.Tests.Shared/UITests.Tests.Shared.projitems

```diff
@@ -19,6 +19,16 @@
         <Compile Include="$(MSBuildThisFileDirectory)TestPageAttribute.cs" />
       </ItemGroup>
     </When>
+    <!-- Have a None Include as well to make all items visible in VS in the Shared Project -->
```

Comment:
Why did you split this up? Please leave this the way it was.

Reply:
The expected logic here is that when these files were included as a part of build you don't need to include them as `None`.

Upon looking here closely, instead of a negative condition, `Otherwise` can also be used!!

Reply:
Would need to re-test this in Visual Studio, they're not there for anything to do with the build. They're literally just to trick VS into showing the files as part of the project, as we're using it as an organizational feature and then splitting out what we need for two different projects.

If this works, it is a bit easier to follow, but it'd be easier to just do as a separate PR later since it's so isolated. Could we revert this part for now, and then we'll get an answer on the AzureRepos package and be able to close this PR out?

Reply:
I've tested the change in VS. It is same as before.